fili-core/src/test/groovy/com/yahoo/bard/webservice/data/DruidResponseParserSpec.groovy

```diff
@@ -404,7 +407,19 @@ class DruidResponseParserSpec extends Specification {
 
         then:
         thrown(UnsupportedOperationException)
+    }
+
+    def "Druid response parser delegates to query for schema columns"() {
+        setup:
+        Stream<Column> columnStream = Mock(Stream)
+        DruidAggregationQuery query = Mock(DruidAggregationQuery)
+        1 * query.buildSchemaColumns() >> columnStream
 
+        when:
+        def result = responseParser.buildSchemaColumns(query)
+
+        then:
+        result == columnStream
```

Comment:
Just make this an `expect`, since it's just "grab the output, and check it", without much other "check the state of the world" aspects to it.